src/main/java/net/sf/jsqlparser/util/deparser/SelectDeParser.java

```diff
@@ -245,6 +249,21 @@ public void visit(Pivot pivot) {
         }
     }
 
+    @Override
+    public void visit(UnPivot unpivot) {
+        boolean showOptions = unpivot.getIncludeNullsSpecified();
```

Comment:
If possible use the visit methods to deparse.

Reply:
I'm not sure how to change this. To me it just looks like how Pivot was implemented.